src/Cli/dotnet/commands/dotnet-workload/install/WorkloadManifestUpdater.cs

```diff
@@ -471,6 +472,17 @@ private IEnumerable<(ManifestId id, ManifestVersion version, SdkFeatureBand feat
                     ManifestVersion manifestVersion;
                     SdkFeatureBand manifestFeatureBand;
                     var parts = manifest.Value.Split('/');
+                    
+                    string manifestVersionString = (parts[0]).ToString();
+                    FXVersion _version;
```

Comment:
You can probably inline the declaration here so you have `TryParse(manifestVersionString, out FXVersion _version)`